src/main/java/seedu/address/logic/commands/event/DeleteEventCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands.event;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Deletes an event from the event list.
+ */
+public class DeleteEventCommand extends Command {
+    public static final String COMMAND_WORD = "delete_event";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the event identified by the index number used in the displayed events list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted Event: %1$s";
+    private final Index targetIndex;
+    public DeleteEventCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
minor nits but I think there should be spacing between the `targetIndex` variable, constructor and the `@Override` annotation

Reply:
good catch!